extern/freitagsfoo

```diff
@@ -46,6 +53,7 @@ sollte nicht länger als 20 Minuten sein.
 
 [https://wiki.chaosdorf.de/index.php?title=Freitagsfoo/${ymd}&amp;action=edit&amp;section=new add your talk here]
 EOF
+my $redir_content = "#REDIRECT [[Freitagsfoo/${redir_date->ymd}]]";
```

Comment:
Unfortunately, this doesn't work in Perl – it needs to be either `'#REDIRECT [[Freitagsfoo/' . $redir_date->ymd . ']]'` or `$redir_date->stftime('#REDIRECt [[Freitagsfoo/%Y-%m-%d]]')`

Reply:
oh, `"#REDIRECT [[Freitagsfoo/${\($redir_date->ymd)}]]"` would also work, but that's really more line noise than readable syntax